container/nginx/njs/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "njs",
+  "type": "module",
```

```suggestion
  "type": "module",
  "private": true,
  "license": "UNLICENSED"
```

to prevent accidental `npm publish`